code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -51,6 +53,17 @@
 	set_light(2, 3, l_color = SSticker.cultdat ? SSticker.cultdat.construct_glow : LIGHT_COLOR_BLOOD_MAGIC)
 
 /mob/living/simple_animal/hostile/construct/death(gibbed)
+	SSticker.mode.remove_cultist(mind, FALSE)
+
+	held_body.forceMove(get_turf(src))
+	SSticker.mode.add_cult_immunity(held_body)
+	if(isliving(held_body))
+		var/mob/living/L = held_body
+		L.key = key
+		L.cancel_camera()
+	held_body = null
+	new /obj/effect/temp_visual/cult/sparks(get_turf(src))
+	playsound(src, 'sound/effects/pylon_shatter.ogg', 40, TRUE)
 	. = ..()
 	SSticker.mode.remove_cultist(mind, FALSE)
```

Comment:
Does there need to be two `SSticker.mode.remove_cultist(mind, FALSE)` calls here?